catalog-rest-service/src/main/resources/json/schema/entity/policies/accessControl/rule.json

```diff
@@ -10,8 +10,14 @@
       "description": "Name that identifies this Rule.",
       "type": "string"
     },
-    "filters": {
```

Comment:
This was ambiguous and causing confusion in both Java pojos and Python pydantic models